NotificationApi/NotificationApi/NotificationApi.csproj

```diff
@@ -1,72 +1,76 @@
-    <Copy SourceFiles="@(ReferencePath)" DestinationFolder="$(OutDir)References" />
-    <Exec Command="$(NSwagExe_Core31) run /runtime:NetCore31 /variables:ProjectPath=&quot;$(MSBuildProjectFullPath)&quot;" />
-    <RemoveDir Directories="$(OutDir)References" />
-  </Target>
+    <PropertyGroup>
+        <UseAppHost>false</UseAppHost>
+        <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+        <DockerComposeProjectPath>..\docker-compose.dcproj</DockerComposeProjectPath>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Azure.Identity" Version="1.3.0" />
+        <PackageReference Include="FluentValidation.AspNetCore" Version="9.3.0" />
+        <PackageReference Include="GovukNotify" Version="4.0.0" />
+        <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.15.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.6" />
+        <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.6" />
+        <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="3.1.6" />
+        <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.15.0" />
+        <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.9.10" />
+        <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.0" />
+        <PackageReference Include="NSwag.AspNetCore" Version="13.9.4" />
+        <PackageReference Include="NSwag.MSBuild" Version="13.9.4">
+            <PrivateAssets>all</PrivateAssets>
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+        <PackageReference Include="Scrutor" Version="3.3.0" />
+        <PackageReference Include="SonarAnalyzer.CSharp" Version="8.14.0.22654">
+            <PrivateAssets>all</PrivateAssets>
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+        <PackageReference Include="VH.Core.Configuration" Version="0.1.12" />
+        <PackageReference Include="ZymLabs.NSwag.FluentValidation" Version="0.4.0" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="..\NotificationApi.Contract\NotificationApi.Contract.csproj" />
+        <ProjectReference Include="..\NotificationApi.DAL\NotificationApi.DAL.csproj" />
+        <ProjectReference Include="..\NotificationApi.Domain\NotificationApi.Domain.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <Content Update="appsettings.json">
+            <CopyToPublishDirectory>Always</CopyToPublishDirectory>
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </Content>
+        <Content Update="appsettings.template.json">
```

Comment:
Can remove this now that template ids are read from the secrets

Reply:
Removed

Reply:
no need to be there for the integration tests to run locally?